src/components/search-page/search-filter-container/SearchFilterContainer.js

```diff
@@ -31,18 +35,45 @@ export function SearchFilterContainer({ filters, filterObj, query }) {
           id="accordion-default"
           aria-label="filter options"
         >
-          {filters?.map((filter, index) => (
-            <SearchFilterSelector
-              key={index}
-              index={index}
-              filter={filter}
+          {isMobile ? (
+            <MobileSearchOptionsContainer
+              filters={filters}
               filterObj={filterObj}
+              query={query}
             />
-          ))}
-
-          <SearchFilterDate index={filters.length} filterObj={filterObj} />
+          ) : (
+            <>
+              <Filters filters={filters} filterObj={filterObj} query={query} />
+            </>
```

Comment:
Are the fragments needed here?
```suggestion
              <Filters filters={filters} filterObj={filterObj} query={query} />
```